src/css/application.css

```diff
@@ -194,7 +194,6 @@ body {
   min-width: 100%;
   max-height: 50vh;
   max-width: 30em;
-  overflow: scroll;
```

Comment:
We do need scrolling for the project picker once you get more than a handful of projects, though—I think this should probably be `overflow: auto`?

Reply:
yep